source/php/Component/Imageinput/Imageinput.php

```diff
@@ -16,5 +16,11 @@ public function init()
         if($filesMax) {
             $this->data['attributeList']['filesMax'] = $filesMax;
         }
+        
+        // Aspect ratio for preview image
+        $defaultAspectRatio = '16:9';
+        $allowedAspectRatios = ['16:9', '4:3', '1:1'];
```

Comment:
I feel like the default and allowed aspect ratios should be setup as class consts rather than defined in the method